nvm.sh

```diff
@@ -1913,11 +1914,13 @@ nvm() {
           if [ $NVM_USE_SILENT -ne 1 ]; then
             echo "Now using system version of node: $(node -v 2>/dev/null)$(nvm_print_npm_version)"
           fi
+          unset NVM_BIN NVM_PATH
           return
         elif nvm_has_system_iojs && nvm deactivate >/dev/null 2>&1; then
           if [ $NVM_USE_SILENT -ne 1 ]; then
             echo "Now using system version of io.js: $(iojs --version 2>/dev/null)$(nvm_print_npm_version)"
           fi
+          unset NVM_BIN NVM_PATH
```

Comment:
this line shouldn't be needed, since you're doing it in `nvm deactivate` already